readme.md

```diff
@@ -72,6 +72,7 @@
 - [Dark Mode with Class](https://github.com/danestves/tailwindcss-darkmode) - Adds `dark` variants based on CSS classes.
 - [CSS Logical Properties](https://github.com/omarkhatibco/tailwind-css-logical-properties) - Generate classnames for CSS Logical Properties for margin, padding, border-width, border-raduis, text-align, float & writing-mode.
 - [CSS Scroll Snap](https://github.com/hawezo/tailwindcss-scroll-snap) - Adds `scroll-snap` utilities.
+- [CSS Alpha Colors](https://github.com/soueuls/tailwind-color-alpha) - Adds alpha colors using your opacity settings.
```

Comment:
Could you change the description to something more explicit, like that for instance? 

```suggestion
+ [CSS Alpha Colors](https://github.com/soueuls/tailwind-color-alpha) - Adds opacity variants to existing colors.
```

Reply:
Looks like a useful plugin, I think a better description would be welcome :)

Reply:
I agree with you and I think your wording is better. I give an example inside my readme I think people would understand the use case.